src/components/common.js

```diff
@@ -65,111 +65,114 @@ const search = function({ wrapperProps = {}, inputProps = {} }) {
-                  height: '3rem', lineHeight: '3rem', backgroundColor: 'white', padding: '1rem',
-                  textAlign: 'center', display: 'block'
-                },
-                Style.elements.pageTitle),
-              href: Nav.getLink('workspaces'),
-              onClick: hideNav
-            }, 'Saturn'),
+          },
+          [
             div({
               style: {
-                padding: '1rem', borderBottom: '1px solid white', color: 'white'
+                boxShadow: '3px 0 13px 0 rgba(0,0,0,0.3)', width: 200,
+                backgroundColor: Style.colors.primary,
+                position: 'fixed', height: '100%'
               }
-            }, [icon('search', { style: { margin: '0 1rem 0 1rem' } }), 'Find Data']),
+            }),
+            div({ style: { width: 200, color: 'white', position: 'absolute' } }, [
+              a({
+                style: _.assign({
+                    height: '3rem', lineHeight: '3rem', backgroundColor: 'white', padding: '1rem',
+                    textAlign: 'center', display: 'block'
+                  },
+                  Style.elements.pageTitle),
+                href: Nav.getLink('workspaces'),
+                onClick: () => this.hideNav()
+              }, 'Saturn'),
+              div({
+                style: {
+                  padding: '1rem', borderBottom: '1px solid white', color: 'white'
+                }
+              }, [icon('search', { style: { margin: '0 1rem 0 1rem' } }), 'Find Data']),
+              div({
+                style: {
+                  padding: '1rem', borderBottom: '1px solid white', color: 'white'
+                }
+              }, [icon('search', { style: { margin: '0 1rem 0 1rem' } }), 'Find Code']),
+              a({
+                style: {
+                  padding: '1rem', borderBottom: '1px solid white', color: 'white',
+                  textDecoration: 'none', display: 'block'
+                },
+                href: Nav.getLink('workspaces'),
+                onClick: () => this.hideNav()
+              }, [
+                icon('grid-view', { class: 'is-solid', style: { margin: '0 1rem 0 1rem' } }),
+                'Projects'
+              ])
+            ]),
             div({
-              style: {
-                padding: '1rem', borderBottom: '1px solid white', color: 'white'
-              }
-            }, [icon('search', { style: { margin: '0 1rem 0 1rem' } }), 'Find Code']),
-            a({
-              style: {
-                padding: '1rem', borderBottom: '1px solid white', color: 'white',
-                textDecoration: 'none', display: 'block'
-              },
-              href: Nav.getLink('workspaces'),
-              onClick: hideNav
-            }, [
-              icon('grid-view', { class: 'is-solid', style: { margin: '0 1rem 0 1rem' } }),
-              'Projects'
-            ])
+              style: { flexGrow: 1, cursor: 'pointer' },
+              onClick: () => this.hideNav()
```

Comment:
Within the containing function wrapper, we need to explicitly call methods that are bound in order to maintain the value of `this` as the instance.